stoobly_agent/app/cli/helpers/openapi_endpoint_adapter.py

```diff
@@ -73,19 +73,32 @@ def adapt(self, spec: Spec) -> List[EndpointShowResponse]:
           endpoint: EndpointShowResponse = {}
           endpoint['id'] = endpoint_counter
           endpoint['method'] = http_method.upper()
-          endpoint['host'] = parsed_url.netloc
+          endpoint['host'] = 'localhost' if parsed_url.netloc == '' else parsed_url.netloc
```

Comment:
I'm unsure about this change. I think OAS lets you not have a host as long as there's a path?

Is a host required for importing into Stoobly? @Jvlythical

Reply:
For now `host` is required